test/decorators.py

```diff
@@ -23,6 +23,20 @@
 from .unit.mock.fake_runtime_service import FakeRuntimeService
 
 
+def production_only(func):
+    """Decorator that runs a test only on production services."""
+
+    @wraps(func)
+    def _wrapper(self, *args):
+        if "dev" in os.environ.get("QISKIT_IBM_URL", ""):
+            raise SkipTest(
+                f"Skipping integration test. {self} is not supported on staging."
+            )
+        func(self, *args)
+
+    return _wrapper
+
```

Comment:
Couldn't find another way to check if the environment was staging - open to suggestions